src/components/Conditions/EvolvingConditionsDetail.jsx

```diff
@@ -45,6 +67,22 @@ const EvolvingConditionsDetail = (props) => {
             <UploadEvolvingConditions />
           </Col>
         </Row>
+        <Row>
+          <Col>
+           <label className="my-4">
+            Items Per Page: 
+            <select
+              className="mx-2"
+              value={itemsPerPage}
+              onChange={(e) => handleItemsPerPageChange(Number(e.target.value))}
+            >
+              <option value={50}>50</option>
+              <option value={100}>100</option>
+              <option value={200}>200</option>
+            </select>
+           </label>
```

Comment:
Would it be possible to add some styling? Either some slight internal padding and rounded edges, or maybe using the [react bootstrap select component](https://react-bootstrap.netlify.app/docs/forms/select/).